auth/gcloud/aio/auth/session.py

```diff
@@ -219,6 +229,21 @@ async def delete(  # type: ignore[override]
             await _raise_for_status(resp)
             return resp
 
+        async def head(  # type: ignore[override]
+            self, url: str,
+            headers: Optional[Mapping[str, str]] = None,
+            timeout: Timeout = 10,
+            params: Optional[Mapping[str, Union[int, str]]] = None,
+            allow_redirects: bool = False,
+        ) -> aiohttp.ClientResponse:
```

Comment:
This is imported as `from aiohttp import ClientResponse as Response`. Perhaps change?

Reply:
Same as the previous comment about wanting to avoid changing unrelated parts of the code.

@TheKevJames might be able to tell you more about this choice.